presentation/src/main/java/bisq/presentation/formatters/TimeFormatter.java

```diff
@@ -59,9 +59,18 @@ public static long getAgeInDays(long date) {
     }
 
     public static String formatAgeInDays(long date) {
-        long ageInDays = getAgeInDays(date);
-        String postFix = ageInDays == 1 ? Res.get("temporal.day") : Res.get("temporal.days");
-        return ageInDays + " " + postFix;
+        long days = getAgeInDays(date);
+        long years = days / 365;
+        long ageInDays = days - years * 365;
+        String daysPostFix = ageInDays == 1 ? Res.get("temporal.day") : Res.get("temporal.days");
+        String dayString = ageInDays + " " + daysPostFix;
+        if (years > 0) {
+            String yearsPostFix = years == 1 ? Res.get("temporal.year") : Res.get("temporal.years");
+            String yearString = years + " " + yearsPostFix;
+            return yearString + ", " + dayString;
+        } else {
+            return dayString;
+        }
```

Comment:
Better than manually handling plurals, can you use [ChoiceFormat](https://docs.oracle.com/javase/tutorial/i18n/format/choiceFormat.html) here?

Reply:
Thanks for the pointer to it. I looked into ChoiceFormat but it feels like a cumbersome API to me. Would rather impl. a custom convenience method to the Res class instead... e.g.
Res.maybePlural("temporal.year", years) which maps in the method depending on years to 
Res.get("temporal.year.none", years) 
Res.get("temporal.year.singular", years) 
Res.get("temporal.year.plural", years) 

The 0 value (none) case is mostly not needed in our case. 
I think such an API would be more convenient to use.

Reply:
See https://github.com/bisq-network/bisq2/pull/1986/commits/d801d1e7a6ad9f5f2915c4c5389b8d9d4cbc52e5